sql-common/client.c

```diff
@@ -1071,6 +1071,10 @@ void mysql_read_default_options(struct st_mysql_options *options,
           break;
         case OPT_enable_cleartext_plugin:
           break;
+        case OPT_bind_address:
+          my_free(options->bind_address);
+            options->bind_address = opt_strdup(opt_arg, MYF(MY_WME));
```

Comment:
this can be `EXTENSION_SET_STRING(options, bind_address, opt_arg);` for consistency with other args right?

Reply:
Sure, but to use the macro the field should be part of st_mysql_options_extention. While I'm not entirely sure, whether this parameter need to be part of the main options or extensions structure, the [comment](https://github.com/MariaDB/server/blob/8d9bc61d0bf783fa792e6c3be37b0eceecbeec89/include/mysql.h#L196) here helped me to decide. What do you think?

Reply:
You made a good choice. Thanks.